moxy/org.eclipse.persistence.moxy/src/test/java/org/eclipse/persistence/testing/jaxb/xmlmarshaller/NoSchemaRefTestCases.java

```diff
@@ -14,24 +14,24 @@
 //     Blaise Doughan - 2.2 - initial implementation
 package org.eclipse.persistence.testing.jaxb.xmlmarshaller;
 
-import jakarta.xml.bind.JAXBContext;
+import org.eclipse.persistence.jaxb.JAXBContext;
 import jakarta.xml.bind.JAXBException;
 import jakarta.xml.bind.ValidationException;
-import jakarta.xml.bind.Validator;
 
 import junit.framework.TestCase;
 
 import org.eclipse.persistence.exceptions.XMLMarshalException;
 import org.eclipse.persistence.jaxb.JAXBContextFactory;
+import org.eclipse.persistence.jaxb.JAXBValidator;
 import org.eclipse.persistence.platform.xml.XMLPlatformException;
 
 public class NoSchemaRefTestCases extends TestCase {
 
     public void testValidateRootNoSchemaReference() {
         try {
             Class[] classes = {Address.class};
-            JAXBContext jc = JAXBContextFactory.createContext(classes, null);
-            Validator validator = jc.createValidator();
+            JAXBContext jc = (JAXBContext) JAXBContextFactory.createContext(classes, null);
```

Comment:
this test should start either with `jakarta.xml.bind.JAXBContext.new...()` or `org.eclipse.persistence.jaxb.XMLBindingContextFactory.createContext(...)`

Reply:
createContext is not static in XMLBindingContextFactory so I changed this to create factory instance first and use it to create context.

Reply:
if it is static, then `new XMLBindingContextFactory().createContext(...)` must work

Reply:
It is working. :) It's just about the way it's written.